README.md

```diff
@@ -297,6 +298,66 @@ When building with CMake, submodules are also recursively cloned automatically.
 can be passed as a CMake flag to disable this functionality. This is useful when you'd like to build CMake while using a
 different commit from a submodule.
 
+### Installation
+
+The C SDK libraries and platform abstractions can be installed to a file system
```

Comment:
How about providing the ability to only install the user provided list of libraries instead of all libraries? Some users may not necessarily want their file system to contain HTTP library if their application only uses MQTT connections